src/containers/Tenant/Diagnostics/TopQueries/getTopQueriesColumns.tsx

```diff
@@ -18,6 +23,21 @@ const TOP_QUERIES_COLUMNS_IDS = {
     ReadRows: 'ReadRows',
     ReadBytes: 'ReadBytes',
     UserSID: 'UserSID',
+    TopQueriesQueryText: 'TopQueriesQueryText',
+    QueryId: 'QueryId',
+};
+
+type QueryColumns = ValueOf<typeof TOP_QUERIES_COLUMNS_IDS>;
+
+const tableColumnsNames: Record<QueryColumns, string> = {
+    CPUTimeUs: 'CPUTimeUs',
+    QueryText: 'QueryText',
+    EndTime: 'EndTime',
+    ReadRows: 'ReadRows',
+    ReadBytes: 'ReadBytes',
+    UserSID: 'UserSID',
+    TopQueriesQueryText: 'QueryText',
+    QueryId: 'QueryId',
```

Comment:
There is no much need to add columns headers to a variable, if there are not used somewhere else. For example, they can be used in Table columns setup control. But here there are used only for headers, so can be just placed in `header` field